bam_plcmd.c

```diff
@@ -276,13 +276,13 @@ static void group_smpl(mplp_pileup_t *m, bam_sample_t *sm, kstring_t *buf,
             const bam_pileup1_t *p = plp[i] + j;
             uint8_t *q;
             int id = -1;
-            q = ignore_rg? 0 : bam_aux_get(p->b, "RG");
+            q = ignore_rg ? NULL : bam_aux_get(p->b, "RG");
```

Comment:
Okay re `NULL`, but the space before `?` is a stylistic choice.  In samtools the existing style is 80% without; in bam_plcmd.c it's exclusively without.  So please leave the existing style here. &lt;/pedantic-picard-hat&gt;